src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -378,6 +390,111 @@
                 return this;
             },
 
+            loadToolbar: function(token) {
+                this.load(
+                    'sfwdt-container',
+                    '{{ path("_wdt", { "token": "xxxxxx" }) }}'.replace(/xxxxxx/gi, token),
```

Comment:
Is this line of code going to work as expected? `path()` is rendered first by Twig in the server and `.replace()` will be executed later by JavaScript in the browser.

Reply:
yeah, and this is expected. Twig will render the URL with `xxxxxx` as token. And then, the JS will replace `xxxxxx` with the token being loaded.

Reply:
Your summary is correct. The `_wdt` path requires the token so we need to provide something to get the path in Twig. Then in JS we replace the placeholder `xxxxxx` with the real token string.